### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by artemkovalyov**
## Context

Automated pipeline for documentation
- Checks if build is successful on Pull request
- Publishes update on push 

I tested this in a forked repo to make sure checks and publishing work.

I added necessary private and public keys to the GH repo

### CODE REVIEW

File: docs/overview/about.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ image:
 ---
 import useBaseUrl from '@docusaurus/useBaseUrl'
 
+## Pipeline test ##
```
Comment (FrankEssenberger) [inline docs/overview/about.md:19]: should this be removed?

File: docusaurus.config.js [hunk 2]
```diff
@@ -2,10 +2,10 @@ module.exports = {
   title: 'SAP Cloud SDK',
   tagline:
     'A one stop shop for developing and extending SAP applications in a Cloud.',
-  url: 'https://github.com/SAP/cloud-sdk',
+  url: 'https://github.com/artemkovalyov/cloud-sdk',
```
Comment (FrankEssenberger) [inline docusaurus.config.js:5]: I guess this was for local testing too?

File: docusaurus.config.js [hunk 3]
```diff
@@ -2,10 +2,10 @@ module.exports = {
...
   baseUrl: '/cloud-sdk/',
   favicon: 'img/favicon.ico',
-  organizationName: 'SAP', // Usually your GitHub org/user name.
+  organizationName: 'artemkovalyov', // Usually your GitHub org/user name.
```
Comment (FrankEssenberger) [inline docusaurus.config.js:8]: see above

File: docusaurus.config.js (hunk 2 above)
Comment (artemkovalyov) [reply docusaurus.config.js:5]: Good catch, I shouldn't have committed this file.

**Pull request closed without merging**